core/src/main/java/com/linecorp/armeria/client/AggregatedResponseAs.java

```diff
@@ -17,16 +17,21 @@
 package com.linecorp.armeria.client;
 
 import java.io.IOException;
+import java.util.function.Predicate;
 
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.HttpStatusClass;
 import com.linecorp.armeria.common.ResponseEntity;
 import com.linecorp.armeria.common.util.Exceptions;
 import com.linecorp.armeria.internal.common.JacksonUtil;
 
 final class AggregatedResponseAs {
+    private static final HttpStatusClassPredicates SUCCESS_PREDICATE
+            = HttpStatusClassPredicates.of(HttpStatusClass.SUCCESS);
```

Comment:
style: Should we place `=` at the end of the first line?
```suggestion
    private static final HttpStatusClassPredicates SUCCESS_PREDICATE =
            HttpStatusClassPredicates.of(HttpStatusClass.SUCCESS);
```